### DISCUSSION TITLE
Resource Pack Help

### DISCUSSION THREAD

**Discussion by BobtheBob123456** [Kas-tle/java2bedrock.sh#323] [Converter Help]
Hi Man,
So remember that pack that I couldn't convert. So I tried fixing it and it still doesn't work.

Can you please tell me what is wrong?

https://www.dropbox.com/s/yx32ks23avtt3h5/April_Java.zip?dl=0

Thanks,

**Comment by BobtheBob123456** [comment]
Hey Man,
Can you please help with this resource pack? I really need it done soon.

Thanks Very Much

**Reply by Kas-tle** [reply]
I'm going to be away from my computer all day and will not have time to look at the pack. For starters though, that's not a direct download link. A direct download links means as soon as the link is clicked, the web browser immediately downloads the file (rather than send the user to a landing page from which the file can then be downloaded).

**Comment by BobtheBob123456** [comment]
Oh sorry, here is a correct version.

https://www.dropbox.com/s/yx32ks23avtt3h5/April_Java.zip?dl=1

Can you help me once you get back?

Thanks

**Reply by Kas-tle** [reply]
Again, this is not a valid resource pack. The assets folder and pack.mcmeta file need to be at the root of the zip. You have instead compressed an enclosing folder above these. As a result, when the converter unzips the pack, there is nothing to find.
![image](https://user-images.githubusercontent.com/26531652/185824933-28323108-a6ae-4dec-a232-acbad5f5e93a.png)

**Comment by BobtheBob123456** [comment]
Ok, so I have made sure that that the pack is in a valid format, yet it still doesn't work.

This time I think some .json was formatted incorrectly but they all look correct to me.

https://www.dropbox.com/s/2y0mk1k7n3kgrvn/April_Java.zip?dl=1

**Reply by Kas-tle** [reply]
If you actually took the time to read the action log you would see that it tells you exactly which file is invalid https://github.com/Kas-tle/java2bedrock.sh/runs/7943755084?check_suite_focus=true#step:6:215

**Reply by BobtheBob123456** [reply]
Oh, I'm stupid. Thanks for the the help.
